### PR TITLE
allow to set different github token.

### PR DESCRIPTION
**Pull request by Mic92**
##### Description

Allow to run ci. Example: https://github.com/nix-community/nix-eval-jobs/pull/28

##### Checklist

- [x] Tested functionality against a test repository (see ["How to test changes"](../README.md#how-to-test-changes))
- [x] Added or updated relevant documentation (leave unchecked if not applicable)

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@ inputs:
     description: 'A space-separated list of inputs to update. Leave empty to update all inputs.'
     required: false
     default: ''
+  token:
+    description: 'GITHUB_TOKEN or a `repo` scoped Personal Access Token (PAT)'
+    default: ${{ github.token }}
```
Comment (cole-h) [inline action.yml:10]: This is what happens right now, correct? Actions use the `github.token` / `$GITHUB_TOKEN`, which doesn't kick off other Actions? (In other words, there won't be a change in behavior, yeah?)

File: action.yml [hunk 2]
```diff
@@ -5,6 +5,9 @@ inputs:
     description: 'A space-separated list of inputs to update. Leave empty to update all inputs.'
     required: false
     default: ''
+  token:
+    description: 'GITHUB_TOKEN or a `repo` scoped Personal Access Token (PAT)'
```
Comment (cole-h) [inline action.yml:9]: All inputs must have a `required` key (https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_idrequired).

File: action.yml (hunk 1 above)
Comment (Mic92) [reply action.yml:11]: Yes. I copied the default value from `peter-evans/create-pull-request@v3` actually

**Review by cole-h — APPROVED** [review]
Thanks!

**Pull request merged**